<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_hello">
      <widget class="QPushButton" name="start">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>360</y>
         <width>150</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exit">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>440</y>
         <width>150</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
      <widget class="QLabel" name="SOKOBAN">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>611</width>
         <height>191</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ravie</family>
         <pointsize>72</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>SOKOBAN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>250</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Choose a map</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>250</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gadugi</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="maxVisibleItems">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="page_game">
      <widget class="QPushButton" name="restart">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>70</y>
         <width>101</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Restart</string>
       </property>
      </widget>
      <widget class="QPushButton" name="menu">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>160</y>
         <width>101</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans MT Condensed</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
